config/application.rb

```diff
@@ -1,14 +1,12 @@
 require File.expand_path('../boot', __FILE__)
 
-# Pick the frameworks you want:
 require 'active_model/railtie'
-# require "active_record/railtie"
 require 'action_controller/railtie'
 require 'action_mailer/railtie'
 require 'action_view/railtie'
 require 'sass'
 require 'sprockets/railtie'
-# require "rails/test_unit/railtie"
+require 'English'
```

Comment:
RuboCop corrects `$~` into `$LAST_MATCH_INFO`, which is coming from the `English` module.